gjkr/member.go

```diff
@@ -0,0 +1,42 @@
+package gjkr
+
+import "threshold.network/roast/ephemeral"
+
+type memberIndex uint16
```

Comment:
Worth dropping a comment that `memberIndex` starts from `1`.

Reply:
See 78a4eaedfbadad89a2d1909e524233713f3acb74, please.